test/quantization/fx/test_quantize_fx.py

```diff
@@ -6158,6 +6159,56 @@ def forward(self, x, weight, bias):
         res = m(*example_inputs)
         self.assertEqual(res, res_ref)
 
+    def test__convert_to_reference_decomposed_fx_per_channel_quant_module(self):
```

Comment:
double underscore

Reply:
yeah this is correct since we are testing `_convert_to_reference_decomposed_fx`